libs/api-test-environment/src/main/java/de/cyface/apitestutils/TestEnvironment.java

```diff
@@ -114,7 +118,8 @@ public TestEnvironment(final Vertx vertx, final VertxTestContext testContext,
 
                     // Set up a Mongo client to access the database
                     final var mongoDbConfiguration = testMongoDatabase.config();
-                    this.mongoClient = EndpointConfig.createSharedMongoClient(vertx, mongoDbConfiguration);
+                    final var dataSourceName = config.getString("data_source_name", DEFAULT_MONGO_DATA_SOURCE_NAME);
+                    this.mongoClient = MongoClient.createShared(vertx, mongoDbConfiguration, dataSourceName);
```

Comment:
This was necessary to make the libs/api-test-environment not dependent of libs/api